tests/data/flamegraph/grey-frames/grey-frames.txt

```diff
@@ -1,5 +1,5 @@
-cksum;_start;__libc_start_main;main;-- 31
 cksum;- 6
+cksum;_start;__libc_start_main;main;-- 31
```

Comment:
Why this ordering change?

Reply:
This is because of the change in flamegraph to sort lines.

Reply:
But this is the _input_ to `flamegraph`, right? Or is that you specifically want to test and see that it sorts? It does seem to change the output of one SVG.

Reply:
I guess this file didn't need to be changed. What happened is the SVG output changed, and I realized it was because it's sorting lines now. I don't know why the original input file wasn't sorted before; it must have been hand-edited. I updated both the SVG file and the input file, but I really only had to change the SVG. The result should be the same either way. I can change it back if you want, though.

Reply:
Nah, I think it's fine -- just wanted to understand the diff. We may want a test along with this change that unsorted and sorted inputs both produce the same SVG though!

Reply:
I pushed a commit that uses the same test SVG for 3 different tests.

1. Multiple inputs, unsorted
2. Single input, unsorted
3. Single input, sorted

This way we know they all produce the same output.